packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -128,7 +131,22 @@ application's performance is enhanced as the operation is performed only once, t
 
 _Methods_
 
--   none
+-   `hideFooter()` - hides the footer section that contains all formatting options and the `footer-actions` slot.
```

Comment:
To hide parts of other controls we use a boolean attribute. See the conventions here: https://github.com/ni/nimble/blob/main/packages/nimble-components/CONTRIBUTING.md#attribute-common-name-patterns

Because showing the footer is probably a reasonable default for a rich text editor generally, then based on the conventions I would expect the footer to be visible by default and then setting the `footer-hidden` attribute to hide the footer.

One thing I'm not completely sure about, wonder if @jattasNI has any thoughts. Should the whole hide footer behavior be the built-in behavior of the rich text editor? Something something progressive disclosure you only see the footer toolbar when it has focus?
Maybe we need footer-visibility: 'auto' (default / undefined), 'visible', 'hidden'
- Auto: hides the footer when not focused, shows the footer when focused
- Visible: forces the footer to always be visible (app can set it to visible if it knows there is unsaved / dirty content)
- Hidden: forces the footer to not be visible (completeness? not sure of the use case)

Reply:
As per the conventions, I have updated it as a `footer-hidden` boolean attribute instead of a method as you mentioned in the first part of the comment.